content/3. Procedurele informatie/4. Techniek/Workshop Git/2. Git Repositories.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 2. Git Repositories
+taxonomie: None
+tags: 
+draft: true
+---
+
+> [!warning] Stuk overslaan?
+> Al een beetje bekend met Git? Dan kun je het eerste deel van deze workshop overslaan en doorgaan vanaf [[3. Git Forks & Branches]]
+
+## Opdracht 1: Repository aanmaken
+>Een repository is de opslagplaats van je project. Hierin staan alle bestanden en de volledige geschiedenis van hun wijzigingen.
+
+**Stap 1:** Log in op GitHub en ga naar de homepagina of je dashboard.
+
+**Stap 2:** Klik op de groene ‘New’ knop aan de linkerkant van de pagina.
+
+> [!info] Tip:
+> Houd de naam van je repository simpel en beschrijvend. Vermijd spaties en speciale tekens (behalve -).
+
+**Stap 2.1:** Geef de repository een naam (bijvoorbeeld FirstRepository) en een beschrijving. Kies ook of je de repository public of private wilt maken.
+
+**Stap 2.2:** Klik op de Create Repository knop.
+
+>[!success] Goed bezig!
+>Je hebt nu je eerste repository aangemaakt. Gefeliciteerd!
+
+  
+## Opdracht 2: Repository clonen
+>Nu je een repository hebt aangemaakt, kunnen we deze clonen naar je PC of laptop. In deze workshop gebruiken we GitHub Desktop. Dit programma maakt het beheren van repositories eenvoudig.
+
+**Stap 1:** Open GitHub Desktop en log in met je GitHub account.
+> [!warning] Let op!
+> Dit moet hetzelfde account zijn waarmee je net de repository hebt aangemaakt.
+
+**Stap 2:** Klik linksboven op de knop Current Repository.
+
+**Stap 3:** Klik op Add en selecteer vervolgens Clone Repository.
+
+**Stap 3.1:** Blijf onder het kopje GitHub.com en zoek onder Your Repositories naar de repository die je zojuist hebt aangemaakt.
+
+**Stap 3.2:** Kies onder Local Path waar je de repository wilt opslaan.
+
+**Stap 3.3:** Klik op Clone.
+	De repository is nu als een map op je PC/laptop opgeslagen, bijvoorbeeld in: C:\Users\Gebruikersnaam\Documents\repos.
+
+>[!success] Gefeliciteerd!
+>Je hebt nu je repository gecloned.
+
+
+## Opdracht 3: Eerste push
```

Comment:
Je behandeld gitignore, maar ik weet zo niet of deze gitignore al standaard meekomt met de repo of het project. zoniet kan je via de cli commando `dotnet new gitignore` heel eenvoudig eentje toevoegen. Of benoem dat er online templates zijn. Misschien is deze niet relevant hoor, dan kan je de comment overslaan